custom_components/tahoma/config_flow.py

```diff
@@ -59,5 +59,12 @@ async def async_step_import(self, import_config: dict):
         if import_config:
             try:
                 return await self.async_validate_input(import_config)
+            except TooManyRequestsException:
+                _LOGGER.error("too_many_requests")
+                return self.async_abort(reason="too_many_requests")
+            except BadCredentialsException:
+                _LOGGER.error("invalid_auth")
+                return self.async_abort(reason="invalid_auth")
```

Comment:
You duplicate this code. You can move it within  _async_validate_input_ 

Reply:
I wanted to just call `self.async_step_user(import_config)` from `async_step_import`, however there are some differences... The async_step_user works with a form and `errors` variable, where the import flow needs to write errors to the logging and `async_abort` the flow.

I couldn't find an elegant way to combine it and I expect the YAML flow to be deprecated in some versions, thus making it easy to remove.